node_ops/node_ops_data.cc

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (C) 2023-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include "db/config.hh"
+#include "gms/gossiper.hh"
+#include "message/messaging_service.hh"
+#include "node_ops/node_ops_data.hh"
+#include "service/storage_service.hh"
+
+#include <seastar/core/sleep.hh>
+#include <seastar/coroutine/parallel_for_each.hh>
+
+static logging::logger nologger("node_ops");
```

Comment:
nit: nologger is a bit too close to "no logger"
since it's static we can just call it `logger`.